templates/web/pages/about.tmpl

```diff
@@ -0,0 +1,142 @@
+{{ define "about" }}
+{{ template "header" . }}
+
+<section class="hero-head">
```

Comment:
Thoughts on making this a background header like the index and hacker pages?  If we do that then we probs don't need that second pic and it can just be the two paragraphs (make sure they're the same width tho)

Reply:
Yea thats a good idea.